pkg/apis/stork/v1alpha1/migrationschedule.go

```diff
@@ -13,8 +13,14 @@ const (
 
 // MigrationScheduleSpec is the spec used to schedule migrations
 type MigrationScheduleSpec struct {
-	MigrationSpec      `json:",inline"`
-	SchedulePolicyName string `json:"schedulePolicyName"`
+	Template           MigrationTemplateSpec `json:"template"`
```

Comment:
any reason we need the additional template field?

Reply:
As mentioned in the commit message
```
Update MigrationSchedule CRD to add Migration under Template.Spec

This is similar to how deployments, statefulsets, etc have pod spec defined
```